### PR TITLE
feat(AI): Update destinations overview image + caption

### PR DESCRIPTION
**Pull request by paperclypse**
The UI was updated. New image to match. Plus slightly more useful context in the caption.

### CODE REVIEW

**Comment by paperclypse** [comment]
@rhetoric101 What do you think about this?

File: src/content/docs/alerts-applied-intelligence/notifications/destinations.mdx [hunk 1]
```diff
@@ -40,19 +40,19 @@ Destination settings require specific [capabilities](/docs/accounts/accounts-bil
...
   alt="Destinations overview page."
   src={destinationsOverview}
 />
 
 <figcaption>
-  **one.newrelic.com > Alerts & AI > Destinations**.
+  Go to **one.newrelic.com > Alerts & AI > Destinations**. Use destinations to choose where your notifications are sent.
 </figcaption>
 
+1. Go to [one.newrelic.com](https://docs.newrelic.com/one.newrelic.com), click **Alerts & AI**, and in the left nav under **Enrich and Respond**, click **Destinations**. The destinations table shows information about the existing destinations and allows users to enable, disable, and modify.
```
Comment (rhetoric101) [inline src/content/docs/alerts-applied-intelligence/notifications/destinations.mdx:53]: @paperclypse, I missed this in the original review, but this first step looks like a couple of steps. You might consider this in a future update:

1. Go to [one.newrelic.com](https://docs.newrelic.com/one.newrelic.com).
2. Click **Alerts & AI**, and in the left nav under **Enrich and Respond**, click **Destinations**.

Also, "allows users" should probably be "allows you."

**Review by rhetoric101 — APPROVED** [review]
@paperclypse, I inserted a few comments you can ignore, but I thought I'd mention them.

File: src/content/docs/alerts-applied-intelligence/notifications/destinations.mdx (hunk 1 above)
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/notifications/destinations.mdx:53]: Thanks! Yeah, this was really just about updating the image for the AI team.

Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/notifications/destinations.mdx:53]: A valid point for sure.

**Pull request merged**